CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * Fixed closing proxy clients
 * Added `ydb.Connection.With(opts ...ydb.CustomOption)` for taking proxy `ydb.Connection` with some redefined options
 * Added `ydb.MetaRequestType` and `ydb.MetaTraceID` aliases to internal `meta` package constants
+* Added `ydb.WithRatelimiterOperationTimeout()` and `ydb.WithRatelimiterOperationTimeout()` options
```

Comment:
* Added `ydb.WithRatelimiterOperationTimeout()` and `ydb. WithOperationCancelAfter()` options

Reply:
ack